### ISSUE TITLE
seeding parameterization

### ISSUE THREAD

**Issue by ShashankBice** [nasa-jpl/autoRIFT#62]
Hi autoRIFT team,
We ([whyjz](https://github.com/whyjz),[dshean](https://github.com/dshean),[MaxVWDV](https://github.com/MaxVWDV),[willkochtitzky](https://github.com/willkochtitzky)) were discussing autoRIFT seeding parameters a while back and meant to clarify this back then, but it spill through my todo list.
If we understand correctly, "Downstream displacement" determines the allowable distance of the search center (can be seeded by pre-existing velocity estimates) and "search distance" determines the fudge or variability around the search center (can be seeded by the value we assume the pre-existing velocity might have changed or vary) ? 

Here is a hand diagram by @dshean explaining the concept.
![image](https://user-images.githubusercontent.com/29011666/175338715-af497ca7-da0f-4416-a665-e73edbfd2d48.png)

Please let us know if we are interpreting this correctly :)
Thanks,
Shashank

**Comment by alex-s-gardner** [comment]
@ShashankBice That's correct. Here are a couple of slides that I presented at ASF last month, maybe their helpful:
![Slide21](https://user-images.githubusercontent.com/32276930/175751831-29409fec-4265-42c1-b78d-f0798024674f.jpeg)
![Slide22](https://user-images.githubusercontent.com/32276930/175751837-b09ab07c-14c9-4f80-91f4-a2cdfe0a54d1.jpeg)
![Slide23](https://user-images.githubusercontent.com/32276930/175751846-40d8035e-20e8-4e4c-97a9-ff1b1b83280a.jpeg)
![Slide24](https://user-images.githubusercontent.com/32276930/175751870-6785e6cf-5add-4b28-8bc9-7e0bed0e7749.jpeg)

**Comment by alex-s-gardner** [comment]
This one might be useful as well
![Slide25](https://user-images.githubusercontent.com/32276930/175751949-47df9c33-f2de-4cd7-b38e-30d04e13968c.jpeg)
:
